legend.html

```diff
@@ -258,7 +248,7 @@ <h3>First aid</h3>
             </tr>
 
             <tr>
-                <td><img src="symbols/openstreetmap-carto/amenity/shower.svg"/></td>
+                <td><img src="symbols/openstreetmap-carto/amenity/shower.svg" height="16"/></td>
```

Comment:
The height should probably be set in CSS https://github.com/cyclosm/cyclosm-cartocss-style/pull/187/files#diff-1040e96f19726619b1ac6a36d00f9aabR7 for easier maintenance.